modules/combined/test/tests/contact_verification/patch_tests/cyl_3/tests

```diff
@@ -156,10 +156,9 @@
     max_parallel = 1
     superlu = true
 
-    # This test is unstable on PETSc 3.7.4 with 64bit indices enabled.
-    dof_id_bytes = 4
     # This test is fails reliably on INTEL compilers with PETSc 3.7.4 too.
```

Comment:
If this fails reliably on Intel, then issue #8200 should not be closed.

Reply:
We would like to go through  if we actually see test fails  with PETSc-3.7.6-intel.  #8200 should be closed, and the intel failing is a different issue. You may open a new issue for this.

Reply:
The comment should be changed, then. What we do know is that PETSc versions between 3.6.4 and 3.7.6 are broken, and because we have at least one problem that fails in 3.7.6, even with those extra tiny pivot options, we still have a problem here.